startup.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import glob
+import os
+import re
+import sys
+
+from sgtk.platform import SoftwareLauncher, SoftwareVersion
+
+
+def _template_to_product_name(template):
+    """
+    Converts a display name template name to a product name.
+
+    For example, "NukeX %s Non-commercial" becomes "NukeX Non-commercial".
+    """
+    return template.replace("%s", "").replace("  ", " ").strip()
+
+
+def _format(template, tokens):
```

Comment:
I've done this to remain Python 2.5 compliant. Should I have bother with the zero config stuff?